web/src/hooks/useNodeSiblings.ts

```diff
@@ -1,4 +1,4 @@
-import { noop } from "~/utils";
+import { noop } from "radashi";
```

Comment:
Not added in this PR, but I would rename this hook as *use-node-siblings*.

Reply:
Right! Thanks pmfor pointing it out. Definitely this is good moment to do so.

Reply:
I went further and removed it since it hasn’t been used for a long time. See https://github.com/agama-project/agama/pull/2412/commits/65564d97ae8ab9f7e6c43a338d481e12df1e456d